packages/go/go-helpers.ts

```diff
@@ -1,16 +1,19 @@
 import tar from 'tar';
 import execa from 'execa';
 import fetch from 'node-fetch';
-import { mkdirp, pathExists, readFile } from 'fs-extra';
+import { mkdirp, pathExists, readFile, remove } from 'fs-extra';
 import { join, delimiter } from 'path';
 import stringArgv from 'string-argv';
-import { debug } from '@vercel/build-utils';
+import { cloneEnv, debug } from '@vercel/build-utils';
+import XDGAppPaths from 'xdg-app-paths';
+import type { Env } from '@vercel/build-utils';
+
 const versionMap = new Map([
-  ['1.19', '1.19.5'],
-  ['1.18', '1.18.1'],
-  ['1.17', '1.17.3'],
-  ['1.16', '1.16.10'],
-  ['1.15', '1.15.8'],
+  ['1.19', '1.19.6'],
+  ['1.18', '1.18.10'],
+  ['1.17', '1.17.13'],
+  ['1.16', '1.16.15'],
+  ['1.15', '1.15.15'],
```

Comment:
I would prefer the version bump be a separate PR

Reply:
My thought with bumping them here is that they will actually get tested, which I have done for 1.18 and 1.19.